Compilers.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27102.0
+# Visual Studio Version 16
```

Comment:
Only change done to Compiler.sln is removing the CodeStyle layer of projects from this solution. Other changes seem to have crept in via the tooling.

Reply:
@dotnet/roslyn-compiler This is the only compiler change in this PR